src/quat.rs

```diff
@@ -128,6 +128,45 @@ macro_rules! impl_quat_methods {
             ))
         }
 
+        /// Gets the minimal rotation for transforming `from` to `to`.
+        /// The rotation is in the plane spanned by the two vectors.
+        /// Will rotate at most 180 degrees.
+        ///
+        /// The input vectors must be finite and non-zero.
+        ///
+        /// `from_rotation_arc(from, to) * from ≈ to`.
+        pub fn from_rotation_arc(from: $vec3, to: $vec3) -> Self {
+            let from = from.normalize();
+            let to = to.normalize();
+            let w2 = (2.0 + 2.0 * from.dot(to)).sqrt();
+            let w2_inv = 1.0 / w2;
+            if w2_inv.is_finite() {
+                let v = from.cross(to) * w2_inv;
+                Self::from_xyzw(v.x, v.y, v.z, w2 / 2.0)
+            } else {
+                // 180° singulary.
+                let tau = 6.28318530717958647692528676655900577;
```

Comment:
didn't quite know how to get in `std::f32::consts::TAU` here in a nicer way, so I just hard-coded it 🤷

Reply:
`$t::consts::TAU`? I think it's pretty recent though, might not be worth bumping MSRV for.

Reply:
If you are using `TAU / 2.0` why not use `PI`?

Reply:
Heh, habits :) I always think of angles in terms of 𝛕

Reply:
I struggled with `$t::consts::TAU` but now realized I needed to change `$t:ty` to `$t:ident`.

I changed this to `std::$t::consts::PI` now